<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cavity db</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>661</width>
     <height>301</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>361</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="columnCount">
    <number>12</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>160</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>160</number>
   </attribute>
   <column>
    <property name="text">
     <string>Cell Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Frequency [MHz]</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Beam pipe r [mm]</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Beam pipe l [mm]</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Iris r [mm]</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Medium r[mm]</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dome r [mm]</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gap [mm]</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sh</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>l_m</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>b_angle</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>beta</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="db_view">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="db_OK">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="db_cancel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_i2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="db_addData">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Cell</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_i2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_i2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cell Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
